.github/workflows/ci-pytorch-test-full.yml

```diff
@@ -29,19 +29,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, windows-2022, macOS-11]
-        python-version: ["3.7", "3.10"]  # minimum, maximum
-        requires: ["oldest", "latest"]
-        release: ["stable"]
-        exclude:
-          # There's no distribution of the oldest PyTorch 1.9 for Python 3.10.
-          # TODO: Remove the exclusion when dropping PyTorch 1.9 support.
-          - {python-version: "3.10", requires: "oldest"}
-        # TODO: re-enable RC testing
-        # include:
-        #   - {os: ubuntu-20.04, python-version: "3.10", requires: "latest", release: "pre"}
-
-    timeout-minutes: 40
+        include:
+          - {os: "ubuntu-20.04", python-version: "3.7", pytorch-version: "1.9", requires: "oldest"}
+          - {os: "ubuntu-20.04", python-version: "3.7", pytorch-version: "1.12"}
+          - {os: "ubuntu-20.04", python-version: "3.8", pytorch-version: "1.9"}  # ex-conda
+          - {os: "ubuntu-20.04", python-version: "3.8", pytorch-version: "1.10"}  # ex-conda
+          - {os: "ubuntu-20.04", python-version: "3.9", pytorch-version: "1.11"}  # ex-conda
+          - {os: "ubuntu-20.04", python-version: "3.9", pytorch-version: "1.12"}  # ex-conda
```

Comment:
Did you add these to checkgroup?

Reply:
I thought I did, but let me check again...